### ISSUE TITLE
DJI assistant 2 developer mode changes override patcher mods?

### ISSUE THREAD

**Issue by Mussemaan** [brett8883/DJI_Super-Patcher#68]
Firstly,
Thanks again for this great patcher. 

Question:
Would like to get more ascend / descent / RTH speed. Apparently only way to get it is 
to use DJI Assistant 2 in developer mode. 
But will changes made with assistant 2 cancel / reset all previously made changes with SP 2.0 ?

**Comment by brett8883** [comment]
> Firstly,
> Thanks again for this great patcher.
> 
> Question:
> Would like to get more ascend / descent / RTH speed. Apparently only way to get it is
> to use DJI Assistant 2 in developer mode.
> But will changes made with assistant 2 cancel / reset all previously made changes with SP 2.0 ?

No. It’s not possible to undo the main patch functions using Assistant 2 even if you wanted to. You would need to flash a stock firmware version to undo the patch. Only the battery mod can be configured with Assistant 2 1.1.2 in debug mode and you’d have to change those specific parameters to do that. Ascent/descent rate is independent of other mods.
